packages/template-retail-react-app/scripts/build-translations.js

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env node
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+/* eslint @typescript-eslint/no-var-requires: "off" */
+
+/**
+ * This script will extract messages from base template and extended app and output all translations in the extended app
+ * If a file is overridden, it won't extract messages from that file in the base template
+ */
+const {exec} = require('child_process')
+const fs = require('fs')
+const path = require('path')
+const packagePath = path.join(process.cwd(), '.', 'package.json')
```

Comment:
Minor: there's no need for a `.`.

```suggestion
const packagePath = path.join(process.cwd(), 'package.json')
```

Reply:
nit, `process.cwd()` returns the dir where you invoke the node process. You could run the process from anywhere, e,g

```sh
node ./build-translations.js
node ../../build-translations.js
```

Maybe a more robust way is to use `__dirname`.

Reply:
@kevinxh In this case, we do want the current working directory. Because we want to grab the package.json from either the base template or the generated project.